plugins/src/android/android.iml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
it is auto generated, do you think we should add it to git ignore?

Reply:
I'm inclined towards yes, if it can be auto-generated again by importing the module, perhaps we don't need to check the project in?

Not sure, guess it's whatever we think will be best for future development & maintenance.

Reply:
(+ .idea folder)

Reply:
Hey @TomBarzilay1 think there's still some files from the .idea folder checked in, can you `git rm` them?

Reply:
Ah sorry, nvm!